src/QirRuntime/test/QIR-static/qsharp/Math.qs

```diff
@@ -28,4 +28,8 @@ namespace Microsoft.Quantum.Intrinsic {
         body intrinsic;
     }
 
-}
\ No newline at end of file
+    operation DrawRandomInt(min : Int, max : Int) : Int {

```

Comment:
I would have expected this operation to be in the Microsoft.Quantum.Random namespace, but it looks like this PR adds it to the Microsoft.Quantum.Intrinsic namespace?

Reply:
Thanks for catching this. Fixed.